### PR TITLE
Handle all htmltools html types

### PR DESCRIPTION
**Pull request by hadley**
@dmurdoch does this make sense? I've slightly generalised the idea to handle any type of html produced by htmltools

### CODE REVIEW

File: tests/testthat/test-evaluate.R [hunk 1]
```diff
@@ -73,3 +73,23 @@ test_that("ansi escapes are translated to html", {
...
+
+test_that("can include literal HTML", {
+  output <- evaluate::new_output_handler(value = identity)
+  env <- env(foo = function() htmltools::div("foo"))
+
+  html <- evaluate_and_highlight("foo()", env = env, output_handler = output, highlight = FALSE)
+  expect_equal(as.character(html), "<span class='r-in'>foo()</span>\n<div>foo</div>")
+})
+
+test_that("captures dependenices", {
```
Comment (dmurdoch) [inline tests/testthat/test-evaluate.R:87]: typo:  should be "captures dependencies"

File: NEWS.md [hunk 2]
```diff
@@ -25,7 +25,9 @@
 * ANSI escapes no longer generate invalid HTML (#79).
 
 * Changes to better support for HTML widgets and rgl in pkgdown 
-  (@dmurdoch, #78).
+  (@dmurdoch, #78). In particular, `evaluate_and_highlight()` now returns
+  an additional attribute containing all extra dependecies needed to render
```
Comment (dmurdoch) [inline NEWS.md:29]: typo:  should be "extra dependencies"

**Comment by dmurdoch** [comment]
I've checked the new code with rgl, and it works fine.  I haven't tried any other htmlwidgets yet.

I marked a couple of typos in NEWS.md and test-evaluate.R .

File: NEWS.md (hunk 2 above)
Comment (hadley) [reply NEWS.md:29]: Dependencies is clearly a tough word for me 😆

**Pull request merged**